container.go

```diff
@@ -30,6 +30,7 @@ import (
 type Process struct {
 	Token string
 	Pid   int
+	URL   string
```

Comment:
I do not believe this belong to the process structure.
I think the URL that we will pass to the shim is a `Pod` URL, although technically it is a proxy one for now. But the shim should not be aware of which entity it's connecting to.

So what we want is to modify the `Pod` structure to replace the `controlSocket` (which we do not use) with a private URL string. We want a getter for that one.
The proxy will give us that URL when registering the VM and potentially when attaching to a running VM and this is when we will fill that `Pod` specific field. This also means that `EnterContainer()` will return a `Pod` pointer as well.